### DISCUSSION TITLE
An option to set behavior of Enter key

### DISCUSSION THREAD

**Discussion by naive231** [brianpetro/obsidian-smart-connections#239] [Ideas]
As a chat window provided by this plugin, I noticed that the prompt I typed wouldn't be sent after press the Enter key. That is not intuitive, especially compared with ChatGPT.

The chat window is support Shift+Enter to send the prompt, that's great. But I still hope that there is an option to toggle behiviours of Enter and Shift+Enter. Of course, if Enter can simply send prompt and Shift+Enter can add a new line by default, that would be great.

**Comment by wwboynton** [comment]
This feels like it would be a pretty quick adjustment and it would be a huge quality-of-life increase. I don't normally reply to +1 threads, but since this one has been dead for nearly two years, I feel it's justified here. I would love to see this behaviour in the config.

**Reply by naivechou** [reply]
Today is 2025/03/31, I think they don't really care about what users' opinions.

**Reply by brianpetro** [reply]
I've explained the issues with the chat input hotkeys in multiple other threads, please use the search function to learn more. 

There is also a new Smart Chat plugin currently in early release testing that may be more amenable for this feature. If so, it will be implemented in that plugin.

PS- if your comment is nothing more than a judgement then it's probably best you keep it to yourself @naivechou 
🌴

**Reply by naivechou** [reply]
Sorry, bro. I should do **more** search if I don't get replied. Thank you for telling us you had explained that issue.
